rewrite-java-test/src/test/java/org/openrewrite/java/ReorderMethodArgumentsTest.java

```diff
@@ -123,15 +108,14 @@ public void test() {
     @Test
     void reorderArgumentsWhereOneOfTheOriginalArgumentsIsVararg() {
         rewriteRun(
-          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(..)",
-              new String[]{"s", "o", "n"}, null, null, null))
-            .cycles(1).expectedCyclesThatMakeChanges(1),
+          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(String,Integer,..)",
+            new String[]{"s", "o", "n"}, null, null, null)),
           java(
             """
               package a;
               public class A {
                  public void foo(String s, Integer n, Object... o) {}
-                 public void bar(String s, Object... o) {}
+                 public void foo(String s, Object... o) {}
```

Comment:
changed this to foo, otherwise we were changing to a non existing method, and this bar was useless.